scripts/cli/bettererResultsToJson.ts

```diff
@@ -0,0 +1,41 @@
+import fs from 'fs';
+
+//@ts-ignore It doesn't seem to import correctly, probably because it doesn end in js or ts
+import resultFile from './../../.betterer.results';
```

Comment:
check reportBettererStats.ts - You should just be able to get the results from that :)

Reply:
Of course 🤦  I literally did that at first